util/src/main/java/com/farao_community/farao/util/SystematicSensitivityAnalysisResult.java

```diff
@@ -26,16 +31,45 @@ public SystematicSensitivityAnalysisResult(Map<State, SensitivityComputationResu
         this.cnecIntensityMap = cnecIntensityMap;
     }
 
-    public Map<State, SensitivityComputationResults> getStateSensiMap() {
-        return stateSensiMap;
+    public Optional<Double> getFlow(Cnec cnec) {
+        return Optional.ofNullable(cnecFlowMap.get(cnec));
     }
 
-    public Map<Cnec, Double> getCnecFlowMap() {
-        return cnecFlowMap;
+    public Optional<Double> getIntensity(Cnec cnec) {
+        return Optional.ofNullable(cnecIntensityMap.get(cnec));
     }
 
-    public Map<Cnec, Double> getCnecIntensityMap() {
-        return cnecIntensityMap;
+    public Optional<Double> getSensitivity(Cnec cnec, State state, RangeAction rangeAction) {
```

Comment:
Actually I think you can remove the state as argument here and take directly the cnec state. Because this could lead to inconsistencies.

Reply:
Done